src/librustc/session/config.rs

```diff
@@ -460,7 +460,7 @@ pub enum PrintRequest {
 
 #[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub enum BorrowckMode {
-    Mir,
+    Body,
```

Comment:
This should also remain `Mir` (basically, if it's an `enum` variant, it should stay `Mir`, because it's referring to `rustc::mir`, e.g. as opposed to `rustc::hir`, not `rustc::mir::Mir`)

Reply:
Ok.

Reply:
This doesn't seem fixed yet, looking at the diff.